shared/remediations/bash/templates/create_services_disabled.py

```diff
@@ -18,7 +18,7 @@
 
 def output_checkfile(serviceinfo):
     # get the items out of the list
-    servicename, packagename = serviceinfo
+    servicename, packagename,daemonname = serviceinfo
```

Comment:
@shawndwells , you are right. The remediation does not need daemonname. However, the remediation Make file uses the same CSV file as OVAL:

services:
	$(SHARED_DIR)/create_services_disabled.py ../../..**/oval/templates/services_disabled.csv**
	$(SHARED_DIR)/create_services_enabled.py ../../../oval/templates/services_enabled.csv

So, without unpacking the third element present in the OVAL CSV file, the remediation py module throws the "too many values" error:
[root@localhost templates]# make services
../../../../../../shared/remediations/bash/templates/create_services_disabled.py ../../../oval/templates/services_disabled.csv
Traceback (most recent call last):
  File "../../../../../../shared/remediations/bash/templates/create_services_disabled.py", line 50, in <module>
    main()
  File "../../../../../../shared/remediations/bash/templates/create_services_disabled.py", line 45, in main
    output_checkfile(line)
  File "../../../../../../shared/remediations/bash/templates/create_services_disabled.py", line 21, in output_checkfile
    servicename, packagename = serviceinfo
**ValueError: too many values to unpack**
make: *** [services] Error 1
 
I am unpacking the daemonname so that this error is not thrown, not using the daemonname anywhere in the py module.